includes/TypesForSolids/ContactSolids.js

```diff
@@ -0,0 +1,149 @@
+if (!RFEM) {
+	throw new Error("This script is only for RFEM, it works with solids.");
+}
+
+/**
+BUG??
+Limit_stress and shear_stiffness cannot be set?? I tried it directly in RFEM console too and no success...
+*/
+
+/**
+* Creates contact solid
+* @class
+* @constructor
+* @param	{Number}	no			Index of contact solid
+* @param	{Array}		solid_list	List of solid indexes
+* @param	{String}	comment		Comment, can be undefined
+* @param	{Object}	params  	Contact solid's parameters, can be undefined
+* @returns	Created contact solid
+*/
+function ContactSolid (no,
```

Comment:
I think better would be to add here parameters which you use in prototype SetContactType and remove parameter solid_list. For assigment, Create prototype function AssingTo(solid_list)